src/test/scripts/nfs/etc/sv/nfs/run

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+modprobe -v nfs
```

Comment:
What was the difference in NFS behaviour when we didn't load the `kmod` and nfs modules into kernel with `modprobe` before?

Are those modules loaded in the host system, i.e. the node running the pod?

Reply:
If another NFS container loaded these modules, or if they are loaded by default on the host system, this is not needed. In Docker for Mac, those modules are not loaded by default on the host VM, so we are loading them here. `modprobe` is a no-op if the module is already loaded, so adding this does not hurt in this case.